MAUI/Cartesian-Charts/SplineRangeArea.md

```diff
@@ -68,6 +68,8 @@ this.Content = chart;
 
 {% endtabs %}
 
+![Spline range area chart type in MAUI Chart](Chart-types-images/maui_spline_range_area_chart.png)
```

Comment:
The type image not have difference from default. change the image.

Reply:
Changed the type image